recipes/pyamg/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "3.0.2" %}
+
+package:
+    name: pyamg
+    version: {{ version }}
+
+source:
+    fn: pyamg-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pyamg/pyamg-{{ version }}.tar.gz
+    md5: bd024253ebae6f7bf2d89fe32fc5d64a
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - zlib  # [linux]
+    run:
+        - python
+        - numpy x.x
```

Comment:
I think it does, but the only way to get it to use setuptools is for numpy to not exist at build time.

I guess this just means it's impossible to use setuptools, and I should remove `` --single-version-externally-managed --record record.txt``, yes?